### PR TITLE
ui: keep selected tab active on reload

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Keeps current tab for documents and persons detailed view on page
  reloads.

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

- selected tabs where lost during reload of page

## How to test?

- select a tab in documents or persons detailed view, select a tab and reload the page or change the language. The selected tab should not change.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

**Comment by Garfield-fr** [comment]
> Nice work, but do we have to put the bundle into persons. This is also used by documents, could we put the bundle in a more general place?

The jquery plugin loaded globally (root bundle) and used locally on persons and documents

File: rero_ils/static/js/rero_ils/rero_keep_tab.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+*/
+
+(function ( $ ) {
+  $.fn.keeptab = function( options ) {
+    var opts = $.extend( {
+      localeItem: 'navkeeptab',
+      selector: '.nav-link'
+    }, options );
+    $('#' + this.attr('id') + ' ' + opts.selector).on('click', function() {
```
Comment (zannkukai) [inline rero_ils/static/js/rero_ils/rero_keep_tab.js:26]: This should be non relevant if `this` doesn't have an `id`. 
Maybe better to use ``` this.find(opts.selected).on(....) ``` ?

Comment (Garfield-fr) [reply rero_ils/static/js/rero_ils/rero_keep_tab.js:26]: I changed this with a new parameter.

File: rero_ils/static/js/rero_ils/rero_keep_tab.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+*/
+
+(function ( $ ) {
+  $.fn.keeptab = function( options ) {
+    var opts = $.extend( {
+      tabsName: '.nav-tabs',
+      localeItemName: 'navkeeptab',
+      selector: '.nav-link'
+    }, options );
+    $(opts.tabsName).find(opts.selector).on('click', function() {
```
Comment (zannkukai) [inline rero_ils/static/js/rero_ils/rero_keep_tab.js:27]: I'm little bit confused with this notation without using `this`.
I think the code below should work too, but shouldn't ; isn't it ?
```
$('#dummy_id').keeptab({ localeItemName: 'document-tab'});
```
Why not simply use `$(this).find(opts.selector).on....` ?

File: rero_ils/static/js/rero_ils/rero_keep_tab.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      var elementId = undefined;
+      if ($(this).attr('id')) {
+        elementId = '#' + $(this).attr('id');
+      } else if ($(this).attr('href')) {
+        elementId = $(this).attr('href')
+      } else {
+        alert('Missing element id or href.');
```
Comment (jma) [inline rero_ils/static/js/rero_ils/rero_keep_tab.js:34]: alert is a bit strong, no?

File: rero_ils/static/js/rero_ils/rero_keep_tab.js (hunk 2 above)
Comment (Garfield-fr) [reply rero_ils/static/js/rero_ils/rero_keep_tab.js:27]: If you have two sections of tabs on same page, you can limit the search area for storage

**Pull request merged**